egs2/wsj0_2mix/enh1/README.md

```diff
@@ -0,0 +1,23 @@
+<!-- Generated by ./scripts/utils/show_enh_score.sh -->
+# RESULTS
+## Environments
+- date: `Wed Jul  1 22:29:51 CST 2020`
+- python version: `3.7.6 (default, Jan  8 2020, 19:59:22)  [GCC 7.3.0]`
+- espnet version: `espnet 0.7.0`
+- pytorch version: `pytorch 1.5.0`
+- Git hash: `e7d5b3a41b32c992b0b5520d02c3aec3343a5f62`
+  - Commit date: `Wed Jul 1 17:55:44 2020 +0800`
+
+config: conf/tuning/train_enh_PSM.yaml
+======= Results in cv_min_8k =======
```

Comment:
This looks ugly
![image](https://user-images.githubusercontent.com/11741550/87365159-8ac70180-c543-11ea-98aa-516249ab5654.png)
Please format it appropriately.

Reply:
We have updated the related scripts, and prettied the `README.md`.